export_template/index.html

```diff
@@ -1,28 +1,24 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Shiny App</title>
+    <title>{{title}}</title>
```

Comment:
Now that we'll be using mustache in the R/Python packages, we should maybe have a default value here?

Reply:
```suggestion
    {{#title}}<title>{{.}}</title>{{/title}}
```

Note no `<title>` if not provided by the `export()` function.